client/build.sbt

```diff
@@ -2,7 +2,7 @@
 
 organization := "com.connectifier.xero"
 name := "client"
-version := "0.13"
+version := "0.14"
 
 // Java. Not Scala
 crossPaths := false
```

Comment:
Github won't let me comment a few lines down, but don't we need to update the location of the schema directory?

sources in (Compile, xjc) += baseDirectory.value / ".." / ".." / "XeroAPI-Schemas" / "src" / "main" / "resources" / "XeroSchemas" / "v2.00"